### PR TITLE
Add SDCFlows summary

### PR DESCRIPTION
**Pull request by effigies**
cc @oesteban to make sure that I'm not lying.

Just saw the note on #2. Can drop the first commit if you want to merge, but might be simplest for you just to copy the bullet points.

### CODE REVIEW

**Comment by poldrack** [comment]
Thanks! just incorporated your sdcflows summary in [3ff9228](https://github.com/poldrack/talks-nipreps/commit/3ff9228e4cca5f22b5c3dfce87de222c08297fc1).  will close this - but @oesteban please leave any comments you have

File: talk/talk.qmd [hunk 1]
```diff
@@ -268,6 +268,31 @@ Ciric et al., 2022, *Nature Methods*
...
 
+- Susceptibility distortion correction is a significant source of analytical flexibility
+    - Multiple fieldmap sequences (spin echo, gradient echo, scanner-reconstructed $B_0$ maps)
+    - Multiple correction implementations with different data structures
+- Comparability among methods is poor, with little insight into sources of variation
+
+- SDCFlows
+    - Defines an intermediate representation of a field inhomogeneity map
+    - Wraps or implements methods for estimating this map from different sequences
+    - Provides a single method for applying estimated maps to EPI sequences
```
Comment (oesteban) [inline talk/talk.qmd:279]:
```suggestion
    - What? Wraps or implements methods for estimating field maps in various scenarios:
        - covering (i) so-called *TOPUP*, (ii) phase-difference fieldmaps and their variants, (iii) *fMRIPrep*'s "fieldmapless"; but
        - it does not cover point-spread-function approaches (extremely marginal).
    - How?
        - Defines a shared representation model (B-Spline) for the field map
        - Technical "nicety": effectively "decouples" estimation and application steps.
    - Why?
        - A single tool can be applied to correct for distortion, no matter how it was estimated
        - The model coefficients can easily be moved with head motion (e.g., across sessions)
        - The model coefficients can easily be integrated in spatial transformation chains (one-shot interpolation)
        - The model coefficients can easily be integrated within other software (e.g., eddymotion)
```

File: talk/talk.qmd [hunk 2]
```diff
@@ -268,6 +268,31 @@ Ciric et al., 2022, *Nature Methods*
...
+
+- SDCFlows is used in fMRIPrep and dMRIPrep, with plans to make a standalone tool
+  for custom processing or easier interoperation with other pipelines
+
+
+## Infrastructure: Architectural redesign
+
+- Problem: fMRIPrep's "one size fits all" design has limitations for emerging use cases
```
Comment (oesteban) [inline talk/talk.qmd:287]: I usually run away from calling this a problem. I believe this is just bringing the future to today (I think I told Russ this "vision" one):

you can see storage, like OpenNeuro, as a place for the original data, which MUST be accessed through filters (except the dataset owners who can access the original data).

Then users may access data through "filters" they are allowed to use. E.g., "defacing & anonymizing" would give you defaced anatomical and remove identifying info. This way, fMRIPrep could become such a filter, and "build the preprocessed dataset" on the spot.

**Comment by oesteban** [comment]
> Thanks! just incorporated your sdcflows summary in [3ff9228](https://github.com/poldrack/talks-nipreps/commit/3ff9228e4cca5f22b5c3dfce87de222c08297fc1). will close this - but @oesteban please leave any comments you have

I just did -- sorry I was slow :D

File: talk/talk.qmd (hunk 1 above)
Comment (oesteban) [reply talk/talk.qmd:279]: sure!

**Comment by oesteban** [comment]
Done!

**Pull request closed without merging**